Sources/ClientRuntime/Networking/Http/SdkHttpRequest.swift

```diff
@@ -70,18 +70,19 @@ extension SdkHttpRequest: CustomDebugStringConvertible, CustomStringConvertible
 }
 
 extension SdkHttpRequestBuilder {
-    public func update(from crtRequest: HTTPRequest, originalRequest: SdkHttpRequest) -> SdkHttpRequestBuilder {
+    public func update(from crtRequest: HTTPRequestBase, originalRequest: SdkHttpRequest) -> SdkHttpRequestBuilder {
```

Comment:
Nit: Would be nice to add docs while we're here

Reply:
++